### PR TITLE
Bump to v1.8.0 

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@vrothberg @mtrmac PTAL

**Review by lsm5 — APPROVED** [review]
LGTM

File: version/version.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the version of the build.
-const Version = "1.7.1-dev"
+const Version = "1.8.1-dev"
```
Comment (TomSweeneyRedHat) [inline version/version.go:4]: I don't think dev should still be here, Just 1.8.0  @mtrmac @vrothberg ?

Comment (TomSweeneyRedHat) [reply version/version.go:4]: Ooops, I was looking at the last commit and skipped over the middle one.  Ignore me.

**Comment by TomSweeneyRedHat** [comment]
LGTM
Can we merge this and cut a v1.8.0 release for Jindrich to use for Monday morning his time?

**Comment by rhatdan** [comment]
Released: https://github.com/containers/skopeo/releases/tag/v1.8.0

**Pull request merged**